lib/rubocop/cop/mixin/minitest_cop_rule.rb

```diff
@@ -24,7 +24,8 @@ module MinitestCopRule
       # @api private
       #
       def define_rule(assertion_method, target_method:, preferred_method: nil, inverse: false)
-        preferred_method = "#{assertion_method}_#{target_method.to_s.delete('?')}" if preferred_method.nil?
+        target_methods = Array(target_method)
+        preferred_method = "#{assertion_method}_#{target_methods.first.to_s.delete('?')}" if preferred_method.nil?
```

Comment:
It can be confusing to see the special treatment of the first element. So, the order of the array shouldn't matter for API user.

Perhaps it would be preferable to require `preferred_method` optional param if more than one element is specified. It might be helpful to add it to the documentation and throw an exception message if it was not specified the option.

Reply:
Thanks! Added a check and updated the docs.